zsh_history_to_fish/command.py

```diff
@@ -74,6 +79,7 @@ def exporter(input_file, output_file, dry_run, no_convert):
     converter = (lambda x: x) if no_convert else naive_zsh_to_fish
     changed = []
     with writer_factory(output_file, dry_run) as writer:
+        i = 0
```

Comment:
I had to add this because otherwise `i` only exists within the for-loop. That broke the script at line 92.